sw/device/lib/base/meson.build

```diff
@@ -37,6 +37,9 @@ sw_lib_mmio = declare_dependency(
 sw_lib_hardened = declare_dependency(
   link_with: static_library(
     'hardened_ot',
-    sources: ['hardened.c'],
+    sources: [
+      'multibits.h',
```

Comment:
Meson automatically locates header files. Is there a reason this is added as a source to the `sw_lib_hardened` target?

I would think you could leave this target un-modified, and just #include the `sw/device/lib/base/multibits.h` wherever its used in SW. Unless I'm missing something here?

Reply:
> Meson automatically locates header files.

FYI: this is not true. We merely have compiler flags configured so that this work. Meson has no idea what headers are.

Reply:
yes, exactly!